esmvaltool/diag_scripts/primavera/eady_growth_rate/eady_growth_rate.py

```diff
@@ -0,0 +1,335 @@
+import logging
+import os
+import sys
+
+import cartopy.crs as ccrs
+import iris
+import iris.analysis
+import iris.cube
+import iris.quickplot as qplt
+import iris.util
+import matplotlib.pyplot as plt
+import numpy as np
+from dask import array as da
+from esmvalcore.preprocessor import (
+    annual_statistics,
+    extract_levels,
+    regrid,
+    seasonal_statistics,
+)
+
+import esmvaltool.diag_scripts.shared
+import esmvaltool.diag_scripts.shared.names as shared_names
+from esmvaltool.diag_scripts.shared import ProvenanceLogger, group_metadata
```

Comment:
Would it make sense to do this the following and then adjust the variables in line 43, 288, 295, 303, and 330 accordingly?

```suggestion
from esmvaltool.diag_scripts.shared import (
    Datasets, 
    group_metadata, 
    names,
    ProvenanceLogger, 
    run_diagnostic
)
```